bin/setup.ts

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+
+import fs from 'fs'
+import util from 'util'
+import path from 'path'
+import { initTarget } from '../src/index'
+
+const fileAccess = util.promisify(fs.access)
+
+async function getTemplatePath() {
+  const pathOptions = [`${__dirname}/../template/package.json`, `${__dirname}/../../../template/package.json`]
+  for (let templatePath of pathOptions) {
+    templatePath = path.dirname(path.normalize(templatePath))
+    if (
+      await fileAccess(templatePath, fs.constants.R_OK)
+        .then(() => true)
+        .catch(() => false)
+    ) {
+      return templatePath
+    }
+  }
+  throw Error(`Failed to find template path`)
+}
+
+async function main(argv: string[]) {
+  if (argv.length < 3) {
+    console.log('setup init')
```

Comment:
Don't you want to console log this when you don't exit?

Reply:
It's just a hint when you have too few args.